example/lib/main.dart

```diff
@@ -20,17 +20,24 @@ class MainPage extends StatefulWidget {
 class _MainPageState extends State<MainPage> {
   int _index = 0;
 
+  final indexStackKey = GlobalKey();
+
+  final page1Key = GlobalKey();
+  final page2Key = GlobalKey();
+  final page3Key = GlobalKey();
+
```

Comment:
I would like to keep the code in Example to the minimum necessary.
If this code is not needed, would it be possible to have it removed along with the print debug(line 73~75, 87~89, 100~102)?